nl_nl.lang

```diff
@@ -177,9 +175,8 @@ gui.xaero_delete_set=Verwijder Set
 gui.xaero_delete_set_message=Weet je zeker dat je de huidige set wilt verwijderen
 gui.xaero_delete_set_message2=De set wordt verwijderd van de huidige wereld.
 gui.xaero_toggle_waypoints=Toggle Waypoints
-gui.xaero_dots_scale=Entity Stip Schaal
-gui.xaero_entity_info=Entity Info
-gui.xaero_toggle_slime=Toggle Slime Chunks
+gui.xaero_dots_size=Entity Stip Schaal
+gui.xaero_toggle_slime=Toggle Slijmkubus Chunks
```

Comment:
Does this make sense or is `Chunks` redundant?

Reply:
The Dutch translator decided to use word "toggle" in several places and phrase "slijmkubus chunks" in `gui.xaero_slime_chunks`, so according to this, "Toggle Slijmkubus Chunks" should be correct.

Reply:
Oh. I just realized that "slime cube" is the name of the mob, so there is no redundancy. Not that calling chunks "cubes" would make sense anyway, but you never know.